### DISCUSSION TITLE
Chair Election 2021-08-04

### DISCUSSION THREAD

**Discussion by robertlong** [omigroup/omigroup#69] [General]
Hey everyone, we'll be hosting our second chair election at our weekly meeting on August 4th at 9:00 AM PDT. You must attend the meeting in Discord in order to vote.

The chair selection process is described [here](https://github.com/omigroup/OMI/blob/main/CHARTER.md#chair-selection)

You can support re-electing the existing chairs or nominate yourselves or others below.

Current Chairs:
Robert
Evo
Jesse

**Comment by Evan-Gold** [comment]
I will be unusually unavailable during this week's meeting, but have been attending every meeting since before the first election.  Should my vote be allowed to, or need to be counted, I vote for the continuing service and leadership of the current chairs.

Evan-Gold AKA Here_comes_the_gold

**Reply by mrmetaverse** [reply]
Thank you for your faith and support Evan-Gold.
